synfig-studio/src/gui/canvasview.cpp

```diff
@@ -3415,9 +3415,8 @@ CanvasView::on_preview_option()
 		RendDesc &r = canv->rend_desc();
 		if(r.get_frame_rate())
 		{
-			float rate = 1/r.get_frame_rate();
-			float beg = r.get_time_start() + r.get_frame_start()*rate;
-			float end = r.get_time_start() + r.get_frame_end()*rate;
+			float beg = r.get_time_start();
```

Comment:
You can go further and remove these variables because they were only used once:
```
po->set_begintime(r.get_time_start());
```

Reply:
makes sense, done :)